context/app/static/js/index.jsx

```diff
@@ -2,9 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import App from './components/App';
+import { setJsonLD } from './schema.org';
 
 ReactDOM.render(
   // eslint-disable-next-line no-undef
   <App flaskData={flaskData} />,
   document.getElementById('react-content'),
 );
+
+/* eslint-disable no-undef */
+if (flaskData?.entity?.entity_type === 'Dataset') {
```

Comment:
Could we do this from the dataset page? Would that make sense? More so a question than a suggestion.

Reply:
My sense is that react code shouldn't have DOM-altering side effects, so I didn't want to push this farther down... So, I'm happy with this, but happy to change, too, if you think that would be better.